app/src/lib/stores/app-store.ts

```diff
@@ -3083,6 +3086,30 @@ export class AppStore extends TypedBaseStore<IAppState> {
     return this._refreshRepository(repository)
   }
 
+  private incrementMergeMetric(source: MergeSource) {
+    switch (source) {
+      // merge from within compare tab
```

Comment:
:art: these comments could live on the enum itself as human-friendly jsdoc comments